### PR TITLE
Improved documentation of supported model types in LTR plugin

### PR DESCRIPTION
**Pull request by JohannesDaniel**
### Description
Enhanced LTR documentation by explicitly mentioning other supported RankLib models and showing RankNet Example

### Issues Resolved
Closes https://github.com/opensearch-project/opensearch-learning-to-rank-base/issues/72

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by epugh — APPROVED** [review]
LGTM

File: _search-plugins/ltr/training-models.md [hunk 1]
```diff
@@ -49,6 +49,29 @@ RankLib outputs the model in its own serialization format. As shown in the follo
 
 Within the RankLib model, each tree in the ensemble examines feature values, makes decisions based on these feature values, and outputs the relevance scores. The features are referred to by their ordinal position, starting from 1, which corresponds to the 0th feature in the original feature set. RankLib does not use feature names during model training.
 
+### Other RankLib models
+
+Notice that RankLib is a library that implements several other model types besides LambdaMART, such as MART,
```
Comment (kolchfa-aws) [inline _search-plugins/ltr/training-models.md:54]:
```suggestion
RankLib is a library that implements several other model types besides LambdaMART, such as MART,
```

File: _search-plugins/ltr/training-models.md [hunk 2]
```diff
@@ -49,6 +49,29 @@ RankLib outputs the model in its own serialization format. As shown in the follo
...
+RankNet, RankBoost, AdaRank, Coordinate Ascent, ListNet or Random Forests. Each of these models has its own set of parameters and training process.
```
Comment (kolchfa-aws) [inline _search-plugins/ltr/training-models.md:55]:
```suggestion
RankNet, RankBoost, AdaRank, Coordinate Ascent, ListNet, and Random Forests. Each of these models has its own set of parameters and training process.
```

File: _search-plugins/ltr/training-models.md [hunk 3]
```diff
@@ -49,6 +49,29 @@ RankLib outputs the model in its own serialization format. As shown in the follo
...
+
+For instance, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```
Comment (kolchfa-aws) [inline _search-plugins/ltr/training-models.md:57]:
```suggestion
For example, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @JohannesDaniel! Just a couple of suggestions.

**Comment by JohannesDaniel** [comment]
@kolchfa-aws Thank you, just committed all of your suggestions.

File: _search-plugins/ltr/training-models.md [hunk 4]
```diff
@@ -49,6 +49,29 @@ RankLib outputs the model in its own serialization format. As shown in the follo
 
 Within the RankLib model, each tree in the ensemble examines feature values, makes decisions based on these feature values, and outputs the relevance scores. The features are referred to by their ordinal position, starting from 1, which corresponds to the 0th feature in the original feature set. RankLib does not use feature names during model training.
 
+### Other RankLib models
+
+RankLib is a library that implements several other model types besides LambdaMART, such as MART,
```
Comment (natebower) [inline _search-plugins/ltr/training-models.md:54]:
```suggestion
RankLib is a library that implements several other model types in addition to LambdaMART, such as MART,
```

File: _search-plugins/ltr/training-models.md [hunk 5]
```diff
@@ -49,6 +49,29 @@ RankLib outputs the model in its own serialization format. As shown in the follo
...
+RankNet, RankBoost, AdaRank, Coordinate Ascent, ListNet, and Random Forests. Each of these models has its own set of parameters and training process.
+
+For example, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```
Comment (natebower) [inline _search-plugins/ltr/training-models.md:57]:
```suggestion
For example, the RankNet model is a neural network that learns to predict the probability of a document being more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```

**Review by natebower — COMMENTED** [review]
@JohannesDaniel @kolchfa-aws Just a few changes. Thanks!

**Pull request merged**